README.md

```diff
@@ -73,12 +59,27 @@ module.**
 * This module does not come with an entity view page for API products at this moment. (See
   [#72](https://github.com/apigee/apigee-edge-drupal/issues/72).) Unless you have a custom module enabled that provides
   this functionality, do not use the "Rendered entity" field formatter on API product entity reference fields.
+  
+## Applying Patches
```

Comment:
I think we can drop this section. We are closing in on a 1.0 release and there shouldn't' be any patches that need to be applied.

Reply:
Unfortunately, this is not up to us. There were and probably there will be Drupal core issues that need to be addressed. We also had problems with other dependencies earlier.
Luckily, after https://github.com/apigee/apigee-client-php/pull/54 merged the module only has special requirements for dev installations _at this moment_.
So I think it is better to provide instructions in-place about the requirements and resolution for possible problems than getting complains when someone bumps into a problem.

Reply:
Yeah I vote to keep the information about patches too. We should probably add the same to apigee_m10n as well.